### DISCUSSION TITLE
No data for Year/Month

### DISCUSSION THREAD

**Discussion by gand07** [bellrichm/weewx-jas#134] [Q&A]
Hi
I'm running version 1.1.0 and I have a question about Year/month.

When I choose a Year and a Month, I have no data for that combination, except the combination for 2023/11.

You can see the site [here](https://argon1.billeder.de/#) 

Any ideas)
Gert

**Comment by bellrichm** [comment]
Looks like you found a bug. I’ll look into it.
Thanks. rich

**Comment by gand07** [comment]
Hi Rich
Thanks for looking. Do You need any further information?
Gert

**Reply by bellrichm** [reply]
I figured out the problem and working on a fix. I’ll let you know if I need anything more.

**Reply by gand07** [reply]
Thanks

**Comment by bellrichm** [comment]
Should be fixed in this [release](https://github.com/bellrichm/weewx-jas/releases/tag/v1.1.1-rc02).

You will need to delete all of the files that WeeWX-JAS generated.

**Comment by gand07** [comment]
Hi
Working perfect.
Thanks
Gert

**Reply by bellrichm** [reply]
Great. Thanks for the help!
